### PR TITLE
Better error handling for creation of demo keys

### PR DESCRIPTION
**Pull request by chris-hamper**
Displays the failure message if the openssl key creation fails during install, which can happen even if the functions are defined

### CODE REVIEW

File: reservoir.install [hunk 1]
```diff
@@ -113,12 +116,16 @@ function _reservoir_generate_keypair() {
     'private_key_type' => OPENSSL_KEYTYPE_RSA
   ));
 
+  if (empty($pk_Generate)) {
+    return [ NULL, NULL ];
```
Comment (wimleers) [inline reservoir.install]: `[ NULL, NULL ]` should be `[NULL, NULL]`

File: reservoir.install [hunk 2]
```diff
@@ -29,8 +29,11 @@ function reservoir_install() {
 
   // Generate initial, temporary keys, for a better onboarding experience.
   if (function_exists('openssl_pkey_new')) {
-    $oauth_settings = \Drupal::configFactory()->getEditable('simple_oauth.settings');
     list($private_key, $public_key) = _reservoir_generate_keypair();
+  }
+
```
Comment (wimleers) [inline reservoir.install]: Let's remove this blank line, so that it's clear the comment on line 30 also applies to this.

**Review by wimleers — CHANGES_REQUESTED** [review]
Minor nits :)

**Comment by chris-hamper** [comment]
@wimleers fixed!

**Review by wimleers — APPROVED** [review]

**Pull request merged**